package.json

```diff
@@ -124,6 +124,24 @@
         "subPermissions": [
           "finance.acquisitions-units-assignments.manage"
         ]
+      },
+      {
+        "permissionName": "ui-finance.transactions.allocations.create",
```

Comment:
IMHO, `transactions` is not needed here, `ui-finance.allocations.create` makes the work.
But it doesn't solve the ticket at all, user will have all the permissions still, `ui-finance.all`. So it should be revised as well

Reply:
Ok. renamed it. Regarding `ui-finance.all` let's wait for PO's answer

Reply:
`ui-finance.all` permissions were updated based on Dennis [coment](https://issues.folio.org/browse/UINV-154?focusedCommentId=78523&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-ortabpanel#comment-78523) and [mod-finance module descriptor](https://github.com/folio-org/mod-finance/blob/master/descriptors/ModuleDescriptor-template.json#L889)